src/components/ModalPortal.js

```diff
@@ -111,11 +112,33 @@ var ModalPortal = module.exports = React.createClass({
   },
 
   afterClose: function() {
+    const { appElement, ariaHideApp } = this.props;
```

Comment:
[Minor] If you're destructing props, I'd suggest to also destruct `onAfterClose` as it's used below.